src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -91,11 +91,11 @@ public partial class DataTransfer
         public Microsoft.AspNetCore.Components.UIDataTransferItem[] Items { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public string[] Types { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
     }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct ElementRef : Microsoft.JSInterop.Internal.ICustomArgSerializer
+    public sealed partial class ElementRef
     {
-        private readonly object _dummy;
-        object Microsoft.JSInterop.Internal.ICustomArgSerializer.ToJsonPrimitive() { throw null; }
+        internal ElementRef() { }
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("_blazorElementRef")]
+        public string Id { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
```

Comment:
This should be `EditorVisibleNever`? Is there anything meaningful a user could do with this?

Reply:
Agreed - hide this if possible. Possibly give it an unfriendly name too. It's unfortunate that we have to make it public.

Reply:
Seeing that you're using `[JsonPropertyName]`, we could definitely give it a name like `__internalId`.